src/bentoml/_internal/utils/formparser.py

```diff
@@ -151,7 +155,10 @@ async def parse(self) -> _ItemsBody:
                         )
                     if b"filename" in options:
                         filename = user_safe_decode(options[b"filename"], charset)
+                        tempfile = SpooledTemporaryFile(max_size=MAX_FILE_SIZE)
```

Comment:
Does this limit the max size of each part?

Reply:
No, this is just the max size stored in memory before it starts writing to disk.